.github/workflows/helm-container-build-push-3rd-gen.yml

```diff
@@ -121,6 +122,109 @@ jobs:
           scout-user: ${{ contains(inputs.scout, 'true') && secrets.DOCKERHUB_USERNAME || '' }}
           scout-password: ${{ contains(inputs.scout, 'true') && secrets.DOCKERHUB_TOKEN || '' }}
 
+  building-container-greenbone-reg:
+    runs-on: self-hosted-generic
+    outputs:
+      digest: ${{ steps.build-and-push.outputs.digest }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Unfortunately we have to format the URL here because in many repos multiple images are built with static URLs.
+      # Later we can also use this step to distribute the images into different OCI projects
+      - name: Create base url
+        id: image-url
+        run: |
+          if echo "${{ inputs.image-labels }}" | grep -q "alpha"; then
+            base_url="$(echo '${{ inputs.image-url }}' | sed 's@greenbone@opensight-dev@g')"
+          else
+            base_url="$(echo '${{ inputs.image-url }}' | sed 's@greenbone@opensight@g')"
+          fi
+          echo "base_url=$base_url" >> $GITHUB_OUTPUT
+
+      - name: Container build and push 3rd gen to opensight
+        id: build-and-push
+        uses: greenbone/actions/container-build-push-generic@v3
+        with:
+          build-context: ${{ inputs.build-context }}
+          build-docker-file: ${{ inputs.build-docker-file }}
+          build-args: ${{ inputs.build-args }}
+          build-secrets: ${{ contains(inputs.build-secret-greenbonebot, 'true') && format('GREENBONE_BOT_PACKAGES_READ_TOKEN={0}', secrets.GREENBONE_BOT_PACKAGES_READ_TOKEN) || inputs.build-secrets }}
+          cosign-key: ${{ secrets.COSIGN_KEY_OPENSIGHT }}
+          cosign-key-password: ${{ secrets.COSIGN_KEY_PASSWORD_OPENSIGHT }}
+          # The tlog function does not currently support an ed25519 key.
+          cosign-tlog-upload: "false"
+          image-url: ${{ steps.image-url.outputs.base_url }}
+          image-labels: ${{ inputs.image-labels }}
+          image-tags: |
+            # create container tag for git tags
+            type=ref,event=tag,value=latest
+            type=match,pattern=v(.*),group=1
+            type=ref,event=pr
+            # use unstable for main branch
+            type=raw,value=unstable,enable={{is_default_branch}}
+          registry: ${{ secrets.GREENBONE_REGISTRY }}
+          registry-username: ${{ secrets.GREENBONE_REGISTRY_USER }}
+          registry-password: ${{ secrets.GREENBONE_REGISTRY_TOKEN }}
+
+      - name: Determine additional url
+        id: determine-url
+        run: |
+          case "${{ inputs.image-url }}" in
+            *"asset-management"*)
+              if echo "${{ inputs.image-labels }}" | grep -q "alpha"; then
+                additional_url="opensight-asset-dev/$(basename ${{ steps.image-url.outputs.base_url }})"
```

Comment:
Wouldn't this lead to `opensight-asset-dev/opensight-dev` in the URL/Path?

Reply:
no, basename greps the last part of the path. 
So it would be e.g. asset-management-backend